tree_hash/src/impls.rs

```diff
@@ -131,30 +128,26 @@ impl TreeHash for U256 {
     }
 
     fn tree_hash_packed_encoding(&self) -> PackedEncoding {
-        let mut result = [0; 32];
-        self.to_little_endian(&mut result);
-        PackedEncoding::from_slice(&result)
+        PackedEncoding::from_slice(self.as_le_slice())
```

Comment:
These are little-endian only methods, I would recommend `to_le_bytes::<32>()`

Reply:
thanks, done in eac7f6353d3ec27e9146ff85e210b01882f54f10